test/int96_timestamp.test.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+const chai = require('chai');
+const assert = chai.assert;
+const { ParquetReader } = require('../lib/reader');
+
+describe('INT96 timestamp handling', function () {
+  this.timeout(30000); // Increase timeout for URL fetching
+
+  const testUrl =
+    'https://aws-public-blockchain.s3.us-east-2.amazonaws.com/v1.0/eth/traces/date%3D2016-05-22/part-00000-54f4b70c-db10-479c-a117-e3cc760a7e26-c000.snappy.parquet';
```

Comment:
Please change this to read in a small reference file (<10-20kb) that is checked into test/test-files, with or without an S3 mock client as in test/reader.js

Reply:
yeah.. i don't have this. i have that file. thousands of them, actually. 🙃

Reply:
well, if you can truncate one of your files and check it into the PR that'd be much appreciated.